### PR TITLE
Fix broken links in the tutorials

### PR DESCRIPTION
**Pull request by jeremyfowers**
The tutorials had some broken links, this PR fixes them.

### CODE REVIEW

File: examples/cli/cache.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ This chapter of the `benchit` CLI tutorials is focused on understanding, inspect
 - [How to keep your filesystem from filling up with build artifacts](#lean-cache)
 
 The tutorial chapters are:
-1. [Getting Started](#https://github.com/groq/mlagility/blob/main/examples/cli/readme.md)
+1. [Getting Started](https://github.com/groq/mlagility/blob/main/examples/cli/readme.md)
 1. [Guiding Model Discovery](https://github.com/groq/mlagility/blob/main/examples/cli/discovery.md): `benchit` arguments that customize the model discovery process to help streamline your workflow.
 1. Working with the Cache (this document): `benchit` arguments and commands that help you understand, inspect, and manipulate the `mlagility cache`.
 1. [Customizing Builds](https://github.com/groq/mlagility/blob/main/examples/cli/build.md): `benchit` arguments that customize build behavior to unlock new workflows.
```
[the remark this answers is not in the document]
Comment (vgodsoe-groq) [reply examples/cli/cache.md:15]: Line 35 and 37 too

File: examples/cli/cache.md [hunk 2]
```diff
@@ -190,7 +190,7 @@ total 20K
 # Thanks!
```
Comment (vgodsoe-groq) [inline examples/cli/cache.md:190]: Dead linke at Line 188 > See the [Lean Cache documentation](https://github.com/groq/mlagility/blob/main/tools_user_guide.md#lean-cache) for more details.

File: examples/cli/discovery.md [hunk 3]
```diff
@@ -7,10 +7,10 @@ This chapter of the `benchit` CLI tutorial is focused on how to guide the tool a
...
-2. Guiding Model Discovery (this document): `benchit` arguments that customize the model discovery process to help streamline your workflow.
-3. [Working with the Cache](https://github.com/groq/mlagility/blob/main/examples/cli/cache.md): `benchit` arguments and commands that help you understand, inspect, and manipulate the `mlagility cache`.
-4. [Customizing Builds](https://github.com/groq/mlagility/blob/main/examples/cli/build.md): `benchit` arguments that customize build behavior to unlock new workflows.
+1. [Getting Started](https://github.com/groq/mlagility/blob/main/examples/cli/readme.md)
+1. Guiding Model Discovery (this document): `benchit` arguments that customize the model discovery process to help streamline your workflow.
+1. [Working with the Cache](https://github.com/groq/mlagility/blob/main/examples/cli/cache.md): `benchit` arguments and commands that help you understand, inspect, and manipulate the `mlagility cache`.
+1. [Customizing Builds](https://github.com/groq/mlagility/blob/main/examples/cli/build.md): `benchit` arguments that customize build behavior to unlock new workflows.
 
 # Model Discovery Tutorials
```
Comment (vgodsoe-groq) [inline examples/cli/discovery.md:16]: Dead link at Line 49 
> See the [Analyze Only documentation](https://github.com/groq/mlagility/blob/main/tools_user_guide.md#analyze-only) for more details.

File: examples/cli/discovery.md [hunk 4]
```diff
@@ -124,7 +124,7 @@ For example, if you run this command:
 benchit benchmark scripts/max_depth.py
 ```
 
-You will get a result that looks very similar to the [Hello World tutorial](#hello-world) tutorial. However, if you peek into `max_depth.py`, you can see that there are two `torch.nn.Linear` modules that make up the top-level model.
```
Comment (vgodsoe-groq) [inline examples/cli/discovery.md:127]: Line 113  See the [Benchmark Multiple Scripts documentation](https://github.com/groq/mlagility/blob/main/tools_user_guide.md#benchmark-multiple-scripts) for more details.

File: examples/cli/discovery.md [hunk 5]
```diff
@@ -176,9 +176,9 @@ You can see that the two instances of `torch.nn.Linear`, `fc` and `fc2`, are ben
```
Comment (vgodsoe-groq) [inline examples/cli/discovery.md:176]: Line 173 See the [Max Depth documentation](https://github.com/groq/mlagility/blob/main/tools_user_guide.md#maximum-analysis-depth) for more details.

File: examples/cli/discovery.md [hunk 6]
```diff
@@ -223,7 +223,7 @@ You can see that both models are discovered, but only `another_pytorch_model` wa
 # Thanks!
```
Comment (vgodsoe-groq) [inline examples/cli/discovery.md:223]: Line 221 > See the [Input Script documentation](https://github.com/groq/mlagility/blob/main/tools_user_guide.md#input-script) for more details.

**Review by vgodsoe-groq — COMMENTED** [review]
Found a few more dead links for you

File: examples/cli/build.md [hunk 7]
```diff
@@ -96,7 +96,7 @@ If we then repeat the `benchit cache stats hello_world_479b1332` we will see tha
 # Thanks!
```
[the remark this answers is not in the document]
Comment (jeremyfowers) [reply examples/cli/build.md:96]: got it!

File: examples/cli/cache.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jeremyfowers) [reply examples/cli/cache.md:15]: got all the https://github.com/groq/mlagility/blob/main/tools_user_guide.md bad links

[the remark this answers is not in the document]
Comment (jeremyfowers) [reply examples/cli/cache.md:15]: and the cache commands one

File: examples/cli/cache.md (hunk 2 above)
Comment (jeremyfowers) [reply examples/cli/cache.md:190]: got it!

File: examples/cli/discovery.md (hunk 4 above)
Comment (jeremyfowers) [reply examples/cli/discovery.md:127]: got it!

File: examples/cli/discovery.md (hunk 5 above)
Comment (jeremyfowers) [reply examples/cli/discovery.md:176]: got it!

File: examples/cli/discovery.md (hunk 6 above)
Comment (jeremyfowers) [reply examples/cli/discovery.md:223]: got it!

File: examples/cli/build.md [hunk 8]
```diff
@@ -59,7 +59,7 @@ The default `Sequence` for CPU and GPU benchmarking performs the following build
 1. Use ONNX ML Tools to the convert the optimized ONNX file to float16
 1. Set the `build_status=successful_build` property
 
-You can see this if you already did the [Hello World tutorial](#hello-world) by running the command:
+You can see this if you already did the [Hello World tutorial](https://github.com/groq/mlagility/blob/main/examples/cli/readme.md#hello-world) by running the command:
```
Comment (vgodsoe-groq) [inline examples/cli/build.md:62]: Line48 Missed Deadlink
> See the [Build Only documentation](https://github.com/groq/mlagility/blob/main/tools_user_guide.md#build-only) for more details.

Comment (vgodsoe-groq) [reply examples/cli/build.md:62]: There's a couple of this file, so maybe search for the guide and replace all the links to it? 
tools_user_guide.md

Comment (vgodsoe-groq) [reply examples/cli/build.md:62]: I think it's just Line 48 and 52 that have the deadlink

**Review by vgodsoe-groq — APPROVED** [review]

File: examples/cli/build.md (hunk 8 above)
Comment (jeremyfowers) [reply examples/cli/build.md:62]: got them, thanks! did a repo-wide search just to be sure that was all of them.

**Pull request merged**